Makefile.am

```diff
@@ -24,4 +24,22 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR
 # IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-SUBDIRS = src tests prof 
+SUBDIRS = src tests prof
+
+# Debian packaging
+dpkg_dir = packaging
+dist_src = $(PACKAGE)-$(VERSION).tar.gz
+deb_src  = $(PACKAGE)_$(VERSION).orig.tar.gz
+
+deb-package:
```

Comment:
can this just depend on the dist target rather than recursively invoking it? 
you could also just use $(distdir).tar.gz for the tarball